src/libs/colors.js

```diff
@@ -17,7 +17,8 @@ const baseColors = {
   danger: '#db3214',
   light: '#e9ecef', // Used as header background color, lightened for background of cells, panels, etc.
   dark: '#333f52', // Used as text color, menu background (lightened), selected background (lightened)
-  grey: '#808080'
+  grey: '#808080',
+  disabled: '#b6b7b8' //Used for disabled components at a 0.5 opacity.
```

Comment:
As of 8c30560af1ea6d6ebc1ca2511aa923556b366638, this comment is no longer accurate.